components/typography/Paragraph.js

```diff
@@ -0,0 +1,26 @@
+
+export default function Paragraph({
+    typeStyle = "body-lg",
+    textColor = "text-gray-700",
+    className,
+    children
+}) {
+
+    let classNames = ''
+    switch (typeStyle) {
+        case 'body-lg': {
+            classNames = `text-lg leading-relaxed font-regular ${className || ''}`
+        }
```

Comment:
When you use switch case you need to remember about break statement. Here you have info why it's so important :)

https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Statements/switch#what_happens_if_i_forgot_a_break

Reply:
Here you should make:

```js
default: { 
  classNames = `text-lg leading-relaxed font-regular ${className || ''}`
}
```

Reply:
Ok check now